README.md

```diff
@@ -94,37 +94,32 @@ To get these credentials please contact one of the maintainers of this project o
 
 We use Docker containers to minimize the installation requirements for development. If you do not have Docker on your system, please follow the following guide: [Install Docker](https://docs.docker.com/engine/install/)
 
-### Scripts
+### Build and Run 
 
-All runtime environment installation and architecture is handled in Docker containers.
+8Knot is a multi-container application. 
 
-Anything not handled by Docker itself (building/networking/parameterizing containers) has
-been handled by the script "scripts/launch\_dev.sh". Please take a moment to become familiar with the steps in this script and suggest any improvements.
+The webserver, worker-pool, and cache containers communicate with one another via docker network.
 
-Please run the following to launch your app:
+All of the build/tear-down is done with docker-compose.
+
+To start the application, run at the top-level of the 8Knot directory:
 
 ```bash
-bash scripts/launch_dev.sh
+docker compose up --build
```

Comment:
Can you add somewhere in the documentation about the install requirements for docker and docker compose?

Reply:
updated above in the "runtime" section